src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$column-width: 100px;
+$gap: 1px;
+
+.calendar {
+  display: flex;
+  width: calc((7 * $column-width) + (6 * $gap));
+  flex-wrap: wrap;
+  gap: $gap;
+  box-sizing: border-box;
+
+  &__day {
+    box-sizing: border-box;
+    border: solid black 1px;
+    background-color: #eee;
+    width: $column-width;
+    height: 100px;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: transform 300ms;
```

Comment:
The `transition` property should include the property you are transitioning, for example, `transition: background-color 300ms, transform 300ms;` to make it clear which properties should have the transition effect.